### PR TITLE
URGENT: changed ms to ks

### PR DESCRIPTION
**Pull request by mvonpapen**
corrected bug: kiloseconds was incorrectly refenced as ms instead of ks

### CODE REVIEW

**Comment by mvonpapen** [comment]
this bug has happened in #136

File: quantities/units/time.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 ks = kilosecond = UnitTime(
     'millisecond',
```
Comment (apdavison) [inline quantities/units/time.py]: This should be 'kilosecond'

Comment (mvonpapen) [reply quantities/units/time.py]: ouch, how did I miss that ;)

**Comment by thisch** [comment]
Thx for fixing it!

**Pull request merged**